universum/modules/launcher.py

```diff
@@ -22,14 +23,14 @@
 ]
 
 
-def make_command(name):
+def make_command(name: str) -> sh.Command:
     try:
         return sh.Command(name)
-    except sh.CommandNotFound as e :
+    except sh.CommandNotFound as e:
         raise CiException(f"No such file or command as '{name}'") from e
 
 
-def check_if_env_set(configuration):
+def check_if_env_set(configuration: configuration_support.ProjectConfiguration) -> bool:
```

Comment:
move to ProjectConfiguration class?

Reply:
Maybe, but not in this PR, that's a separate change.